libraries/aws_iam_user.rb

```diff
@@ -35,7 +37,7 @@ def access_keys
   end
 
   def to_s
-    "IAM User #{@name}"
+    "IAM User #{@user[:name]}"
```

Comment:
Can we introduce a method like `name` that reads out the user name? This would allow anybody to access the value for a test too